app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<p style="color: green"><%= notice %></p>
+
+<% @public_recipes.includes(:user, :recipe_foods).each do |recipe| %>
+  <div class="row">
+    <div class="card d-flex flex-row">
+      <div class="row col-6">
+        <h2>
+          <%= recipe.name %>
+        </h2>
+        <span>By: <%= recipe.user.name %></span>
+      </div>
+      <div class="d-flex flex-column">
+        <p>Total food items: <%= recipe.recipe_foods_count %></p>
+        <p>Total price: <%= calculate_total_price(recipe) %></p>
+      </div>
+    </div>
+  </div>
```

Comment:
- [ ] Kindly make sure that when clicking on a public recipe item, it should lead to the recipe details as well to follow the project requirement.
![Screenshot from 2024-01-26 17-41-25](https://github.com/Surafels/recipe-app/assets/91142432/1ebe4b78-f41d-4bcf-9384-3839c484eff2)